FALSEHOODS.md

```diff
@@ -112,7 +112,7 @@ surprising how many false assumptions programmers continue to make about them.
 
 1.  **Phone numbers contain only digits**
 
-    In Israel, certain advertising numbers start with a `*`.
+    In Israel, certain advertising numbers start with a `*`. In New Zealand, non-urgent traffic incidents can be reported by calling      `*555` from a mobile phone.
```

Comment:
If you can just fix the alignment here and remove the extra spaces after "calling" we can just merge this in directly. Thanks!